c/utils.c

```diff
@@ -34,6 +34,8 @@
 #include "debug.h"
 #include "printables_for_dump.h"
 #include "timeutls.h"
+#include "logging.h"
```

Comment:
`utils.c` should be zowelog free. This is utils, so they're reused by everyone, you don't want to make everyone link with logging, le, etc. 

I think we should make sure that the functions that need to print error information can return that error info instead. For example, the SLH code will return NULL if it can't allocate data.